sdks/java/core/src/test/java/org/apache/beam/sdk/PipelineTest.java

```diff
@@ -418,7 +418,7 @@ public CompositeBehavior enterCompositeTransform(Node node) {
   }
 
   /**
-   * Tests that {@link Pipeline#replaceAll(List)} throws when one of the PTransformOverride still
```

Comment:
This test now becomes no-op.

Reply:
That is true, but I want to keep this test, since it shows that its valid to replace a transform and return the same transform